kuma/feeder/tests/test_utils.py

```diff
@@ -143,7 +143,7 @@ def test_fetch_feed_unchanged(hacks_feed, mocked_parse, enabled):
     """If the ETag matches, the feed is enabled but not processed."""
     hacks_feed.last_modified = datetime(2018, 2, 26, 21, 23, 38)
     hacks_feed.etag = 'W/"1da1fc6a456fd49c32a9291b38ec31ee-gzip"'
```

Comment:
This line that stood out to me, so I double-checked. In Python 3, ``feedparser.parser`` returns with a string ``etag``, not a byte stream, so we should be OK. There may be other issues, but not this :)